slick-testkit/src/main/scala/com/typesafe/slick/testkit/tests/NewQuerySemanticsTest.scala

```diff
@@ -268,7 +268,7 @@ class NewQuerySemanticsTest extends AsyncTest[RelationalTestDB] {
       c <- coffees.filter(_.price < 800).map((_, 1))
     } yield (c._1.name, c._1.supID, c._2)
 
-    val a5 = seq(
+    def a5 = seq(
```

Comment:
To get useful logs when an execution fails. Otherwise the several other queries have already been compiled in between.